jdbc/src/main/java/software/amazon/timestream/jdbc/TimestreamDriver.java

```diff
@@ -175,9 +176,47 @@ public boolean jdbcCompliant() {
      * @return the name of the currently running application.
      */
     private static String getApplicationName() {
-        // Currently not supported.
-        // Need to implement logic to get the process ID of the current process, then check the set of running processes and pick out
+        // What we do is get the process ID of the current process, then check the set of running processes and pick out
         // the one that matches the current process. From there we can grab the name of what is running the process.
+        try {
```

Comment:
Also, need to update the documentation to explain how application nae is being collected. Provide example collected data on Linux, macOS and Windows.

Reply:
Document is updated.